config/datocms/migrations/1741855116_addGroupingBlock.ts

```diff
@@ -1,12 +1,20 @@
 import { Client } from '@datocms/cli/lib/cma-client-node';
 
 export default async function (client: Client) {
+  
+  //@ts-expect-error rich_text_blocks is only available on Modular Content fields
+  const homeBodyBlocks = (await client.fields.find('home_page::body_blocks')).validators.rich_text_blocks?.item_types;
+  //@ts-expect-error rich_text_blocks is only available on Modular Content fields
+  const pageBodyBlocks = (await client.fields.find('page::body_blocks')).validators.rich_text_blocks?.item_types;
+  //@ts-expect-error rich_text_blocks is only available on Modular Content fields
+  const pagePartialBlocks = (await client.fields.find('page_partial::blocks')).validators.rich_text_blocks?.item_types;
```

Comment:
Nice to see that you can get everything easily from the client!

_I don't fully understand why it's inside `rich_text_blocks` though; is this how DatoCMS structures fields inside blocks?_
The above was what I was originally planning on asking, but I see that DatoCMS indeed uses this structure when creating blocks too!

```
await client.fields.create(groupingItem.id, {
   ...
    validators: {
      rich_text_blocks: {
        item_types: [
```